src/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>WazaCode To do List Project</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+
+  </head>
+<body>
+    <h1>Welcome to WazaCode To DO List App</h1>
+    <div class="header">
+      <h1>Today's To Do</h1>
+      <button id="refreshButton">
+        <i class="fas fa-sync-alt"></i>
+      </button>
+    </div>
+    <div id="app">
+        <textarea id="newTaskInput" class="newTaskInput" placeholder="Add to your list"></textarea>
+        <ul id="todoList">
+    </div>
+        <p class="clearCompletedText" id="clearCompletedText">Clear all completed tasks</p>
+  <script src="index.js"></script>
+</body>
```

Comment:
- **I struggled to find the Adding and deleting button in your App. Can you make these buttons visible? because we need to check if the functions you implemented are working** 
- **Your App**
![Capture](https://github.com/ishimwepati/To-Do-list/assets/63972670/5ab9a4f1-7602-4f11-8e66-ab0b1dd861c6)
- **Provided App**

![Capture1](https://github.com/ishimwepati/To-Do-list/assets/63972670/3766cc27-289c-4733-8edb-e54b61fcd2df)

Reply:
Hello! 

My side it working perfectly, down the GIF to explain

https://github.com/ishimwepati/To-Do-list/assets/14818338/f1e0c3a0-e941-4f4e-9ef2-b6c38e4b75a8